core/migrate/data.go

```diff
@@ -72,4 +72,13 @@ var migrations = []migration{
 		DROP INDEX signers_type_id_idx;
 		DROP INDEX assets_sort_id;
 	`},
+	{Name: `2017-06-28.0.core.coreid.sql`, SQL: `
+		CREATE TABLE core_id (
+			singleton boolean DEFAULT true NOT NULL,
+			id text,
+			CONSTRAINT core_id_singleton CHECK (singleton)
+		);
```

Comment:
might need to run `go run cmd/dumpschema/main.go` to pick up the constraint in core/schema.sql